test/parallel/test-tls-inception.js

```diff
@@ -59,11 +61,13 @@ a.listen(common.PORT, function() {
       rejectUnauthorized: false
     });
     ssl.setEncoding('utf8');
+    var buf = '';
     ssl.once('data', function(data) {
-      assert.equal('hello', data);
+      buf += data;
       gotHello = true;
     });
```

Comment:
Arghh... It should be `on` indeed. @santigimeno would it be interesting to you to submit a PR with a fix for this?